COT/ovf.py

```diff
@@ -503,6 +503,43 @@ def system_types(self, type_list):
         XML.set_or_make_child(system, self.VIRTUAL_SYSTEM_TYPE, type_string)
 
     @property
+    def product(self):
+        """Short descriptive product string (XML ``Product`` element)."""
+        if self.product_section is not None:
+            return self.product_section.findtext(self.PRODUCT, None)
+        return None
+
+    @product.setter
+    def product(self, product_string):
+        if self.product_section is None:
+            self.product_section = self.set_or_make_child(
+                self.virtual_system, self.PRODUCT_SECTION)
+            # Any Section must have an Info as child
+            self.set_or_make_child(self.product_section, self.INFO,
+                                   "Product Information")
```

Comment:
Now that we have four separate places where we might need to create PRODUCT_SECTION, I think it's time this got pulled out into a helper method - something like `def create_product_section_if_missing(self)`. Would you be willing to do this as part of your diffs?